docs/README.md

```diff
@@ -0,0 +1,624 @@
+#     "volume": 68.85441
+#   },
+#   {
+#     "card": "sig_debit",
+#     "date": "2018-02-01",
+#     "volume": 2.41721
+#   },
+#   {
+#     "card": "credit",
+#     "date": "2018-03-01",
+#     "volume": 58.09631
+#   },
+#   {
+#     "card": "sig_debit",
+#     "date": "2018-03-01",
+#     "volume": 5.42187
+#   }
+# ]
+```
+
+### Aggregation
+
+Abstract aggregation operation.
+    
+
+<details markdown='1'><summary>Aggregations</summary>
+
+#### None
+
+
+
+#### 3 Month Moving Average
+
+Periods = 3, Frequency = Month
+    
+
+#### 6 Month Moving Average
+
+Periods = 6, Frequency = Month
+    
+
+#### 12 Month Moving Average
+
+Periods = 12, Frequency = Month
+    
+
+#### 18 Month Moving Average
+
+Periods = 18, Frequency = Month
+    
+
+</details>
+
+### Attributes
+
+Attribute base class.
+Metrics are columns of interest (Ex. Volume, Net Revenue, Processing Cost) that are co-tabulated with
+attribute columns (Ex. Date, Card, Region) that are used to filter or group the metric.
+
+<details markdown='1'><summary>Attributes</summary>
+
+#### Card
+
+**Card** is an attribute of central importance in the AIM system.
+
+There are 5 basic card types:
+- credit
+- signature_debit aka *sig_debit*
+- pin_debit
+- opt_blue
+
+And 2 non-basic card types:
+- bank_cards (credit + sig_debit)
+- other_cards
+
+The metrics coming from raw processor data which are reported on individual
+card types may be filtered and grouped by card types and are referred to as
+"card metrics" as opposed to "non-card metrics".
+
+#### Average Ticket Tier
+
+A merchant's ticket tier is based on its **average** number of transactions (or "tickets")
+over a rolling 12 month period.
+
+#### Annual Volume Tier
+
+A merchant's volume tier is based on its **total** volume over a rolling 12 month period.
+    
+
+#### Region
+
+Geographic region of the transaction.
+Canada is a region.
+
+#### State
+
+U.S. State of the transaction
+
+#### ZIP
+
+Zip code of the transaction
+
+#### MSA
+
+City of the transaction
+
+#### Sales Model
+
+Sales model code
+
+#### Industry Classification Type
+
+Industry Classification Type. Currently this is either MCC or SIC.
+    
+
+#### Industry Group
+
+Hierarchical grouping of Industries
+    
+
+#### Industry
+
+Industry the merchant belongs to.
+    
+
+#### Portfolio
+
+A grouping of merchants within an organization.
+    
+
+#### Data Month
+
+Date is one of the AIM required attributes.
+Traditionally date has been by month due to month being the frequency of the aim
+application, though other aggregation levels are possible and may show up in the
+future. The term era is used to denote a chunk of time. Ex. The month of June, as
+opposed to June 1.
+
+#### Standalone Merchants
+
+Binary on if the merchant is part of a chain or not.
+
+#### Vintage
+
+Year merchant entered the market
+
+</details>
+
+### Metrics
+
+Base metric class.
+    
+
+<details markdown='1'><summary>Metrics</summary>
+
+#### COS Total Processing Fees
```

Comment:
Al:
> The metrics listed should follow a logical flow like the harmonized fields or be in alphabetical order
but there is no logical order as they are listed below.
There should possibly be examples for each type of value/field that would go into the following
metrics

Reply:
We can do alphabetical.